build/db.mk

```diff
@@ -0,0 +1,118 @@
+# ------------------------------------------------------------
+# Copyright 2023 The Radius Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#    
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------
+
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=ucprulz!
+POSTGRES_CONTAINER_NAME=radius-postgres
+
```

Comment:
is the password here because this is v1 of postgres addition?

Reply:
This is for running a local PostgreSQL container for testing. This will replace where we currently store data in Kubernetes `radius-testing` in dev/test workflows.